frb_dart/lib/src/env/stub.dart

```diff
@@ -0,0 +1,3 @@
+// see https://github.com/flutter/flutter/issues/55870
+
+export 'io.dart' if (dart.library.html) 'web.dart';
```

Comment:
nit: Looks like we can remove stub.dart, because currently it is:

* env.dart: export stub or io or web
* stub: export io or web

change to:

* env.dart: export io or web
* stub: removed

Reply:
refactored in 4c064a6